src/main/java/org/motechproject/mots/constants/MotsConstants.java

```diff
@@ -3,4 +3,6 @@
 public final class MotsConstants {
   public static final String SIMPLE_DATE_FORMAT = "yyyy-MM-dd";
 
+  public static final String UUID_PATTERN =
```

Comment:
if it's not used anywhere else move it to UuidValidator

Reply:
ok